source/Plugins/Language/Swift/SwiftArray.cpp

```diff
@@ -87,6 +87,9 @@ SwiftArrayNativeBufferHandler::SwiftArrayNativeBufferHandler(
   auto opt_size = elem_type.GetByteSize(process_sp.get());
   if (opt_size)
     m_element_size = *opt_size;
+  auto opt_stride = elem_type.GetByteStride(process_sp.get());
+  if (opt_stride)
+    m_element_stride = *opt_stride;
```

Comment:
If you failed to get a byte stride, don't you want to error out of the array formatter?  It seems very unlikely to work in that case.

Reply:
*Nod*. I'll make this change on stable.